expat/tests/acc_tests.c

```diff
@@ -239,69 +227,64 @@ START_TEST(test_accounting_precision) {
        "<!ENTITY % e2 '%e1;'>\n"
        "%e1;\n",
        "\xEF\xBB\xBF<!ATTLIST doc a1 CDATA 'value'>" /* UTF-8 BOM */,
-       strlen("\xEF\xBB\xBF<!ATTLIST doc a1 CDATA 'value'>"), filled_later},
+       strlen("\xEF\xBB\xBF<!ATTLIST doc a1 CDATA 'value'>")},
+      {"<!DOCTYPE r [\n"
+       "  <!ENTITY five SYSTEM 'first.ent'>\n"
+       "]>\n"
+       "<r>&five;</r>",
+       "\xEF\xBB\xBF" /* UTF-8 BOM */, NULL, 0},
```

Comment:
A rebase went wrong here I assume, this test was moved upwards in #781 and adding it here would make us have it twice, please revert the addition.

Reply:
Whoops. Yes, rebase mistake!